### PR TITLE
Publish CVE-2021-41043

### PR DESCRIPTION
**Pull request by infrastation**

### CODE REVIEW

**Comment by cve-team** [comment]
infrastation is not recognized currently as an account participating in the Git Pilot. To ensure timely processing of your pull request, please contact the CVE Team using either the CVE Request web form (https://cveform.mitre.org/) or an email (cve@mitre.org) and confirm your identity.

**Comment by infrastation** [comment]
Apparently, the suggested e-mail address is **not** a valid point of contact:
> This email address is not monitored.
> To contact the CVE Program, use https://cveform.mitre.org and choose “Other” from the dropdown menu.

File: 2021/41xxx/CVE-2021-41043.json [hunk 1]
```diff
@@ -4,14 +4,65 @@
     "data_version": "4.0",
```
Comment (cve-team) [inline 2021/41xxx/CVE-2021-41043.json:4]: MISC or CONFIRM, but name != url for reference.

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

File: 2021/41xxx/CVE-2021-41043.json [hunk 2]
```diff
@@ -4,14 +4,63 @@
...
```
Comment (cve-team) [inline 2021/41xxx/CVE-2021-41043.json:4]: An exception of type JSONDecodeError occurred. Arguments:

Comment (cve-team) [inline 2021/41xxx/CVE-2021-41043.json:4]: ('Expecting property name enclosed in double quotes: line 64 column 13 (char 1900)',)

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

**Comment by infrastation** [comment]
Since the script is not flagging any more issues, I suggest to merge this pull requests right away. Alternatively, someone could fix the broken script so it re-validates the file again, and see what the results are.

**Comment by infrastation** [comment]
It looks like this repository is not actually manned by any member of MITRE staff.

**Comment by infrastation** [comment]
Oh, thank you!

**Pull request merged**